storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -237,10 +237,6 @@ QtObject {
         }
     }
 
-    function getNetworkIcon(chainId) {
```

Comment:
General remark, not strictly related to this PR - ideally we should keep stubs empty, they are needed only to overwrite types of stores from the app. If we need actual mocks shared among multiple storybook pages, we should create mocks derived from the stub and use them in pages. In that approach we can use different mocks for different pages and tests. Putting mocking stuff directly into a stub forces all pages/tests to use it.

Reply:
aaah gotcha! I can keep that in mind for next time!

Reply:
btw where would we keep these shared derived stores in this case?

Reply:
Hey, regarding stubs I wouldn't care too much in this PR, bc stubs are barely touched here
but in general I would opt for creating some kind of mocks directory and keep there mockes inheriting form stub's types but then in the code it's important to have a possibility to pass that store to component, the problem is when components instantiate stores on their own, then injecting dependency is impossible, needs to be refactored first.

Generally we don't have too many pages requiring stores, but if it's the case store is mocked in-place directly in page, e.g.: status-desktop/storybook/pages/CommunitiesPortalLayoutPage.qml
but if mock is bigger and common for several pages then I agree that it would be nice to have it in a separate file.